package.json

```diff
@@ -4,32 +4,48 @@
   "description": "Proxy that handles interactions with pagoPA",
   "main": "index.ts",
   "repository": "https://github.com/teamdigitale/italia-pagopa-proxy",
-  "author": "Federico Feroldi <federico@teamdigitale.governo.it>",
+  "author": "Sergio Manetta <sergio.manetta@it.ibm.com>",
```

Comment:
well, thanks :D

Reply:
Just to save you from my responsibilities :P